cico_setup.sh

```diff
@@ -10,4 +10,34 @@ build_image() {
     make docker-build
 }
 
+tag_push() {
+  local target=$1
+  local source=$2
+  docker tag ${source} ${target}
+  docker push ${target}
+}
+
+push_image() {
+    local image_name
+    local image_repository
+    local tag
+    local push_registry
+    image_name=$(make get-image-name)
+    image_repository=$(make get-image-repository)
+    tag=$(git rev-parse --short=7 HEAD)
+    push_registry="push.registry.devshift.net"
+
+    # login first
+    if [ -n "${DEVSHIFT_USERNAME}" -a -n "${DEVSHIFT_PASSWORD}" ]; then
```

Comment:
Good catch :) I copy+pasted these lines from https://github.com/fabric8-services/fabric8-auth/blob/master/cico_setup.sh#L117-L121, but I forgot to rename the variable.